### ISSUE TITLE
Remove OPAL password not acknowledged

### ISSUE THREAD

**Issue by alfechner** [Drive-Trust-Alliance/sedutil#96]
I tried to remove my OPAL encryption by following [this guide](https://github.com/Drive-Trust-Alliance/sedutil/wiki/Remove-OPAL).

However the command returns a `NOT_AUTHORIZED` status code. After all I used exactly the same password than for booting the device at startup which works absolutely fine. So I guess it should be correct. The command in question is 

```
sudo ./sedutil-cli --revertnoerase  'my-password' /dev/sda
```

. Note, that my password includes special chars `!` and `%` which are gone if I don't use single quotes. 

I also tried to disable the encryption following [this guide](https://github.com/Drive-Trust-Alliance/sedutil/wiki/Encrypting-your-drive). The first command

```
sedutil-cli -–disableLockingRange 0 <password> <drive> 
```

gave me the same error code.

Help is needed here.

**Comment by alfechner** [comment]
After several restarts it worked fine. I always used the same password. Somehow weired, happy that it worked out in the end. For me there is some bug going on. Leave it to you guys to close this issue. Since I cannot tell how to reproduce this problem it's up to you if you consider me just another guy who typed in his password incorrectly several times ;)

**Comment by r0m30** [comment]
Since you can't tell me how to reproduce this I don't know how I would debug it.

**Comment by alfechner** [comment]
A bit of time passed and I faced the exact same problem again. I'm still using the same password including special chars `!` and `%`. 

I again used single quotes and got an authentication error. I fiddled around a bit and ended up using `$(printf %b 'password')` which also *didn't* work out.

I rebooted the system (hard drive didn't shut down) and tried again. Still didn't work.

I restarted the computer (hard drive shut down and locked) and tried again. Success.

I did this because I stumbled upon the error constant `AUTHORITY_LOCKED_OUT` in the [FAQ](https://github.com/Drive-Trust-Alliance/sedutil/wiki/FAQ). The description says:

> You have exceeded the maximum number of tries to enter your password correctly, the drive must be powered down before you can continue

Might it be possible that I reached the number of maximal tries fiddling around with the escaping problem? In this case the system should return `AUTHORITY_LOCKED_OUT` instead of `NOT_AUTHORIZED`.
